openfga/v1/openfga_service.proto

```diff
@@ -1046,7 +1046,7 @@ message ReadChangesRequest {
   ];
 
   string type = 2 [(validate.rules).string = {
-    pattern: "^[^:#\\s]{,254}$"
+    pattern: "^[^:#\\s]{0,254}$"
```

Comment:
Type is optional in ReadChanges API https://openfga.dev/api/service#/Relationship%20Tuples/ReadChanges

Reply:
If it's optional it should also have a validation rule that allows that. A regex allowing an empty string doesn't cover our bases fully.

Use the string 'ignore_empty' validation rule in conjunction with this.